server/routes/api/last-reads.js

```diff
@@ -0,0 +1,58 @@
+const router = require("express").Router();
+const { LastRead } = require("../../db/models");
+
+// get lastReads for a given conversation
+router.get("/:conversationId", async (req, res, next) => {
+  try {
+    
+    const { conversationId } = req.params;
+    const lastReads = await LastRead.findReadsByConversation(
+      conversationId
+    );
+
+    res.json(lastReads);
+
+  } catch (error) {
+    next(error);
+  }
+});
+
+// update current user lastRead for a given conversation
+router.post("/", async (req, res, next) => {
+  try {
+    if (!req.user) {
+      return res.sendStatus(401);
+    }
+    const userId = req.user.id;
+    const data = req.body;
+
+    const { conversationId, messageId } = data.find(read => userId === read.userId);
+
+    let lastRead = await LastRead.findLastRead(
+      conversationId,
+      userId
+    );
+    // if lastRead exists update it
+    if (lastRead) {
+
+      await LastRead.update({ messageId }, { 
+        where: { id: lastRead.id }
+      });
+
+      // return with new messageId too
+      lastRead.messageId = messageId;
```

Comment:
Some comments about your implementation:
- Are you sure that POST is the best request method to update the database?
- The route should be protected against users not part of the conversation
- There are some improvements in your database implementation - mentioned in the main comment above.